package/scripts/presto_coordinator.py

```diff
@@ -39,17 +42,22 @@ def start(self, env):
         Execute('{0} start'.format(daemon_control_script))
         if 'presto_worker_hosts' in host_info.keys():
             all_hosts = host_info['presto_worker_hosts'] + \
-                host_info['presto_coordinator_hosts']
+                        host_info['presto_coordinator_hosts']
         else:
             all_hosts = host_info['presto_coordinator_hosts']
-        smoketest_presto(
-            PrestoClient('localhost','root',
-                         config_properties['http-server.http.port']),
-            all_hosts)
+        # sleep some seconds for waiting coordinator start finished
+        time.sleep(30)
```

Comment:
OK, I have seen it. To add this code just because some errors will be raised (just like ExecutionFailed mentioned blew) . And the retries will be interrupted, Never retry again. 
Add some seconds for sleeping will skip the ExecutionFailed exception in most cases.